### ISSUE TITLE
e621 not working

### ISSUE THREAD

**Issue by serveral1** [Bionus/imgbrd-grabber#2855]
**Bug description**

As of some hours ago, e621.net as a source seems to have stopped working. From what I understand it seems like the upped their cloudflare security or something after the site went offline for some hours and then after coming back Grabber just doesn't seems to work anymore with it.

**Steps to reproduce**

1. Select e621.net as source
2. Search any tag
3. "Possible reason: server offline error" returned

**Expected behavior**

For e621 as a soruce to work.

**Context**

[main.log](https://github.com/Bionus/imgbrd-grabber/files/10200912/main.log)

**Screenshots**

**System information**

- OS: Windows 10
- Grabber version: 7.9.1 x64

**Additional context**

In all honesty, this just seems to be more of the same cloudflare conundrum that just keeps growing and growing as more booru sites opt to hire cloudflare for their protection services. I don't want to sound too pessimistic but the way things are currently shaping up for  Grabber means that it's going to eventually loose it's original purpose of serving as a tool for ripping content from boorus (or at least the ones using cloudflare)

**Comment by serveral1** [comment]
So looking a little bit more into it, the cause of this bug seems to be some part of some left over jank from their recent maintenance (the reason the site was down) since some users are reporting API issues within the site itself and other projects that interact with the e621 API also reporting issues. I'm hoping it's just something temporary really and doesn't need to be addressed yet.

**Comment by Bionus** [comment]
Let's hope that's the case 😄

**Comment by GameCharmer** [comment]
Hit this same issue with The Wolf's Stash on Android.  The app had to open up a browser so that cloudflare could verify/validate/authorize/whatever the connection.  Basically a quick "prove you're a human" browser check.  I've tried connecting via Chrome on Windows which did give me the cloudflare page for a few seconds before allowing me in, but it looks like it's tied specifically to that browser.  When opening in Firefox, I got the same page for a few seconds before proceeding.  The only thing I could think of at this point that would help would be some kinda browser within grabber that could open a window, wait for cloudflare, then grab whatever cookies are being set.

After connecting in browser, check your cookies.  There should be a `cf_clearance` cookie.  I tried bringing this into grabber with no luck.
